platforms/software/dependency-management/src/integTest/groovy/org/gradle/integtests/resolve/BeforeResolveIntegrationTest.groovy

```diff
@@ -73,14 +73,14 @@ task copyFiles(type:Copy) {
         outputContains('[dep1-1.0.jar, dep2-1.0.jar]')
 
         when:
-        executer.expectDocumentedDeprecationWarning("Mutating the dependencies of configuration ':conf' after it has been resolved. This behavior has been deprecated. This will fail with an error in Gradle 9.0. After a Configuration has been resolved, consumed as a variant, or used for generating published metadata, it should not be modified. Consult the upgrading guide for further information: https://docs.gradle.org/current/userguide/upgrading_version_8.html#mutate_configuration_after_locking")
+        executer.expectDocumentedDeprecationWarning("Mutating a Configuration after it has been observed. This behavior has been deprecated. This will fail with an error in Gradle 9.0. The dependencies of configuration ':conf' were mutated after the configuration was resolved. After a Configuration has been resolved, consumed as a variant, or used for generating published metadata, it should not be modified. Consult the upgrading guide for further information: https://docs.gradle.org/current/userguide/upgrading_version_8.html#mutate_configuration_after_locking")
```

Comment:
❓ What does "observed" mean?  Observed by who or what?  I think this isn't clear what caused the problem, which makes it unclear how to avoid it.

Reply:
This is described later in the deprecation message in the advice section.

I switched the order of some text in the main message and the advice to make it more clear.